### PR TITLE
Feature/mustache lambda documentation

### PR DESCRIPTION
**Pull request by MichalFoksa**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`. If contributing template-only or documentation-only changes which will change sample output, be sure to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) first.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.1.x`, `5.0.x`. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR
Follow up PR to #3368.

PR documents common lambdas in [templating](https://openapi-generator.tech/docs/templating).

Please have a look into `Mustache Lambdas` section. Rest of changes are removed trailing spaces by my editor.

### CODE REVIEW

**Comment by macjohnny** [comment]
can you please merge the current master?

**Comment by MichalFoksa** [comment]
Here you are, PR rebased to current master.

File: docs/generators/openapi-yaml.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ sidebar_label: openapi-yaml
 |ensureUniqueParams|Whether to ensure parameter names are unique in an operation (rename parameters that are not).| |true|
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
 |prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
-|outputFile|output filename| |null|
+|outputFile|output filename| |openapi/openapi.yaml|
```
Comment (macjohnny) [inline docs/generators/openapi-yaml.md]: this file is generated and shouldn't be modified directly

Comment (MichalFoksa) [reply docs/generators/openapi-yaml.md]: Can you PLS tell me where is it generated and how?

Comment (MichalFoksa) [reply docs/generators/openapi-yaml.md]: The thing is that outputFile default value in documentation is not correct. it should be `openapi/openapi.yaml`.

Comment (macjohnny) [reply docs/generators/openapi-yaml.md]: i think it should be changed here:
https://github.com/OpenAPITools/openapi-generator/blob/9e391efd1d1f2c62ad9b8b9348d650f72062e3d3/modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/ConfigHelp.java#L55-L57

Comment (macjohnny) [reply docs/generators/openapi-yaml.md]: default value: https://github.com/OpenAPITools/openapi-generator/blob/1bf8c15206c290e68051f65cc6fe1e1ed76ecb6d/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/OpenAPIYamlGenerator.java#L39

Comment (macjohnny) [reply docs/generators/openapi-yaml.md]: and make sure to run `./bin/utils/export_docs_generators.sh` and commit the changed file

Comment (MichalFoksa) [reply docs/generators/openapi-yaml.md]: I see, CLI option is not fully described in OpenAPIYamlGenerator.java. It is missing default value declaration.

Comment (MichalFoksa) [reply docs/generators/openapi-yaml.md]: Here you are. Commit-ed openapi-yaml.md is the one created by `./bin/utils/export_docs_generators.sh`.

Comment (macjohnny) [reply docs/generators/openapi-yaml.md]: great, you found the right file, I just noticed my suggestion which file to change was wrong ;-)

**Comment by wing328** [comment]
@MichalFoksa thanks for the PR, which has been included in the 4.1.0 release: https://twitter.com/oas_generator/status/1160000504455319553

**Comment by MichalFoksa** [comment]
@wing328 You are absolutely welcome! Cogratulation to release.

**Pull request merged**